modules/format/main/index.coffee

```diff
@@ -1,62 +1,78 @@
 
 siSuffixes = ['y','z','a','f','p','n','µ','','','K','M','G','T','P','E','Z','Y']
 
-zeroPad = (number, pad) ->
+zeroPad = (number, { length }) ->
```

Comment:
Should there be a default for this? Or maybe this one should be non optional?

Reply:
Decided to go with a default. If this length should be non-optional, the digits/sf should also be non-optional